src/pwacompat.js

```diff
@@ -124,7 +150,25 @@ function unused() {
     return (part) => part || '';
   }
 
+  /**
+   * Adds an element in the <head> if it's not present already
+   * nb: we check, but this won't override any _earlier_ (in DOM order)
+   * @param {string} localName tag name
+   * @param {*} attr key-value collection of attributes
+   */
   function push(localName, attr) {
+    if (localName === 'meta' && isMetaPresent(attr.name)) {
+      return;
+    }
+    // in case of links, comparing either href or sizes (because it's used for icons too)
+    if (
+      localName === 'link' &&
+      (isLinkPresent('href', attr.href) ||
+        (attr.sizes && isLinkPresent('sizes', attr.sizes))
+      )
+    ) {
+      return
```

Comment:
nit: missing semicolon

Reply:
done ;) You'll have to excuse me. I'm so used to Vue.js "style" (no semicolons, yolo, hahaha 😉)